src/utils/feedData.js

```diff
@@ -74,72 +72,48 @@ export function mapTransactionsHistory(
       };
     });
 
-  if (keepHashDuplicatesIfBetweenAccounts) {
-    const accountsAddresses = accounts.map((acc) => getAccountAddress(acc));
-    const ascendingHistory = orderBy(concatedHistory, ['createdAt'], ['asc']);
-
-    const historyWithTrxBetweenAcc = ascendingHistory.reduce((alteredHistory, historyItem) => {
-      const { from: fromAddress, to: toAddress, hash } = historyItem;
-      const isTransactionFromUsersAccount = accountsAddresses
-        .some((userAddress) => addressesEqual(fromAddress, userAddress));
-      const isTransactionToUsersAccount = accountsAddresses
-        .some((userAddress) => addressesEqual(toAddress, userAddress));
-      const eventWithSameHashExists = alteredHistory.some((item) => item.hash === hash);
-
-      if (eventWithSameHashExists) {
-        if (isTransactionFromUsersAccount && isTransactionToUsersAccount) {
-          return [...alteredHistory, {
-            ...historyItem,
-            accountType: getAccountTypeByAddress(toAddress, accounts),
-            isReceived: true,
-            betweenAccTrxDuplicate: true,
-            _id: `${historyItem._id}_duplicate`,
-            createdAt: historyItem.createdAt + 1,
-          }];
-        }
-        return alteredHistory;
-      } else if (duplicatePPN) {
-        const itemTag = get(historyItem, 'tag');
-        if (itemTag && itemTag === PAYMENT_NETWORK_ACCOUNT_TOPUP) {
-          const duplicate = {
-            ...historyItem,
-            smartWalletEvent: true,
-            _id: `${historyItem._id}_duplicate`,
-            createdAt: historyItem.createdAt - 1,
-          };
-          return [...alteredHistory, duplicate, historyItem];
-        }
-        return [...alteredHistory, historyItem];
+  if (!keepHashDuplicatesIfBetweenAccounts) return orderBy(concatedHistory, ['createdAt'], ['desc']);
+
+  const accountsAddresses = accounts.map((acc) => getAccountAddress(acc));
+  const ascendingHistory = orderBy(concatedHistory, ['createdAt'], ['asc']);
+
+  const historyWithTrxBetweenAcc = ascendingHistory.reduce((alteredHistory, historyItem) => {
+    const { from: fromAddress, to: toAddress, hash } = historyItem;
+    const isTransactionFromUsersAccount = accountsAddresses
+      .some((userAddress) => addressesEqual(fromAddress, userAddress));
+    const isTransactionToUsersAccount = accountsAddresses
+      .some((userAddress) => addressesEqual(toAddress, userAddress));
+    const eventWithSameHashExists = alteredHistory.some((item) => item.hash === hash);
+
+    if (eventWithSameHashExists) {
+      if (isTransactionFromUsersAccount && isTransactionToUsersAccount) {
+        return [...alteredHistory, {
+          ...historyItem,
+          accountType: getAccountTypeByAddress(toAddress, accounts),
+          isReceived: true,
+          betweenAccTrxDuplicate: true,
+          _id: `${historyItem._id}_duplicate`,
```

Comment:
Just curious about the `_` here?

Reply:
Related to moving pieces, do not want to touch this in this PR as it will increase the scope.